### PR TITLE
LPS-168833 Replace liferay-ui:icon with clay:icon in asset-publisher-web

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
[Link to ticket](https://issues.liferay.com/browse/LPS-168833)

## Test Scenarios - general steps

* Open the application menu on the top and go to Control Panel > Configuration > System Settings.
* Look for “openoffice” and click on the result “OpenOffice Integration".
* Activate Server Enabled and save.
* Navigate to Content & Data > Web Content and create 3 new Basic Web Content.
* Go to Site Builder > Pages and create a new public page.
* Add an Asset Publisher and enter into the configuration.

## Test Scenario 1

* Go to Set-up > Asset Selection and select "manual". 
* In Asset Entries, select the web contents that you created in step 1.
* Check the up and down buttons in the right side of the web contents selected.
ASSET ENTRIES

## Test Scenario 2

* Go to Set-up > Asset Selection and select "dynamic". 
* Go to Set-up > Ordering and check the buttons to select the order (ascending and descending in both cases)
Order By
And Then By

## Test Scenario 3

* Go to Set-up > Asset Selection > Scope and check the delete button for the site.
Current Site (New Ste)

## Test Scenario 4

* Go to Set-up > Display Settings > Set and Enable and go to the tab "Enable conversion to".
* Hover in the icon with the interrogation to check it.
ENABLE CONVERSION T

## Test Scenario 5

* Go to Set-up > Asset Selection and select "dynamic", save and publish the page. 
* View the page and click on one of the titles.
* Check the back button next to the title.
Test Test

### CODE REVIEW

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/ordering.jsp [hunk 1]
```diff
@@ -45,23 +45,23 @@
 		String orderByType1 = assetPublisherDisplayContext.getOrderByType1();
 		%>
 
-		<div class="d-inline-flex order-by-type-container">
-			<liferay-ui:icon
+		<div class="d-inline-flex lfr-portal-tooltip order-by-type-container">
+			<clay:button
```
Comment (sandrodw3) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/ordering.jsp:49]: Um, the button looks a bit disaligned after the changes. I think you should either remove `monospaced` (it wasn't monospaced before) or add a div that wraps both the select and the buttons with `d-flex align-items-center`

**Review by sandrodw3 — COMMENTED** [review]
Looks super good!! 🥳 I only left a small suggestion about aligning the button, please tell me if I can help you 🤍

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/ordering.jsp [hunk 2]
```diff
@@ -23,8 +23,8 @@
 		md="6"
 	>
 		<div class="h5"><liferay-ui:message key="order-by" /></div>
```
Comment (sandrodw3) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/configuration/ordering.jsp:25]: Nice commit 😂

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/10999

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**